oshdb-util/src/test/java/org/heigit/bigspatialdata/oshdb/util/celliterator/IterateByContributionRelationsTest.java

```diff
@@ -37,10 +37,13 @@
    * {@link GridOSHRelations}.
    */
   public IterateByContributionRelationsTest() throws IOException {
-    osmXmlTestData.add("./src/test/resources/different-timestamps/polygon.osm");// read osm xml data
-    areaDecider = new OSMXmlReaderTagInterpreter(osmXmlTestData);// Used to provided information
+    // read osm xml data
+    osmXmlTestData.add("./src/test/resources/different-timestamps/polygon.osm");
+    // Used to provided information
+    areaDecider = new OSMXmlReaderTagInterpreter(osmXmlTestData);
     // needed to create actual geometries from OSM data
```

Comment:
:thinking: I guess this comment should be appended to the comment which was previously after the `;` at the end of the line above:

> Used to provide information needed to create actual geometries from OSM data

(also there seems to be a typo in there: _provided_ -> _provide_)